pkg/virt-controller/services/template.go

```diff
@@ -372,14 +372,20 @@ func (t *templateService) renderLaunchManifest(vmi *v1.VirtualMachineInstance, i
 		return i.Slirp != nil
 	})
 	if len(slirpIfaces) > 0 {
+		var slirpSidecarImage string
 		if plugin := ReadNetBindingPluginConfiguration(t.clusterConfig.GetConfig(), SlirpNetworkBindingPluginName); plugin == nil {
-			return nil, fmt.Errorf("couldn't find %s network binding plugin configuration, make sure its specified in Kubevirt config", SlirpNetworkBindingPluginName)
+			// In case no Slirp network binding plugin is registered (i.e.: specified in in Kubevirt config) use default image
```

Comment:
This file is long enough, I prefer to keep this logic in the network specific file (even if it means the event manager has to be passed one step deeper).

Reply:
Done